core/src/main/java/com/graphhopper/routing/weighting/GenericWeighting.java

```diff
@@ -124,40 +119,16 @@ public double calcWeight(EdgeIteratorState edgeState, boolean reverse, int prevO
         if (time == Long.MAX_VALUE)
             return Double.POSITIVE_INFINITY;
 
-        int spatialId = gEncoder.getSpatialId(edgeState.getFlags());
```

Comment:
Would you mind to add applying the speed and accessibility changes and recover may some tests that I removed? This seems indeed important to me, now that you pointed me to it